docs/release_docs/Prod-ci-cd_ReleaseFlow.md

```diff
@@ -0,0 +1,24 @@
+## PROD-CI-CD
+
+Prod-CI-CD workflow can be use or trigger by Pushing code into main branch.
```

Comment:
Re-frame it as: "Prod Workflow is triggered automatically whenever any PR is merged into the `main` branch. The workflow builds the applications and deploys the changes to the RF Platform Production environment."